.env.sample

```diff
@@ -27,3 +28,9 @@ DISCORD_CLIENT_ID=clientId
 DISCORD_CLIENT_SECRET=clientSecret
 # the url accessed after the Discord authorization processed is fulfilled
 DISCORD_REDIRECT_URI=http://localhost
+
+# MerkleDistributor overrides
+MERKLE_DISTRIBUTOR_CHAIN_ID=
+MERKLE_DISTRIBUTOR_ADDRESS=
+REFERRALS_START_WINDOW_INDEX=
```

Comment:
I primarily added this to make the offset to determine which claims to consider for the rewards reset configurable. E.g. only consider `Claimed` events starting from window index `1`. Maybe the intent gets clearer [here](https://github.com/across-protocol/scraper-api/pull/111/files#diff-76c276375bb5aa6c7b38880fb43207dc2240004d7a7dc4cb9f1619a10d7aaa10R22). 

But now that I think about it, we would need to re-run a migration if that env var changes. So maybe this isn't a valid use case anymore. 

I also thought this could be good for testing purposes where we could shift windows for different rounds of Airdrop + Referrals claims in single MD deployment. E.g.
- 1st testing round Airdrop window index = 0, referrals start index = 1
- 2nd testing round referrals start index = 2
- 3rd round official QA airdrop = 3, referrals = 4